source/myukfast/accounts.md

```diff
@@ -0,0 +1,27 @@
+# Accounts
+
+```eval_rst
+   .. title:: MyUKFast | Accounts
+   .. meta::
+      :title: MyUKFast | Managing Contacts and Permissions | UKFast Documentation
+      :description: How to perform various Accounts related activities in MyUKFast
+      :keywords: ukfast, accounts, invoice, invoice query, credit note, billing, payment method, payment, commission, remittance, refund, VAT
+```
+
+## Invoicing
+UKFast provides all information that you should require relating to invoicing, recurring and payments within MyUKFast, with a number of specific pages available within the [Billing](https://my.ukfast.co.uk/billing/accountsview.php) section of MyUKFast.
+
+To view the relevant information, once you are logged into MyUKFast, navigate to the following sections:
+
+1. [View invoices and credit notes](https://my.ukfast.co.uk/billing/accountsview.php) - **Billing > Transactions**
+2. [View recurring costs](https://my.ukfast.co.uk/billing/recurringcosts.php) - **Billing > Recurring Costs**
+3. [How to pay](https://my.ukfast.co.uk/billing/how-to-pay) - **Billing > How to pay**
+4. [Managed stored credit/debit cards](https://my.ukfast.co.uk/billing/cards/index.php) - **Billing > Credit/Debit Cards**
+5. [Invoice Queries](https://my.ukfast.co.uk/billing/invoice-queries) - **Billing > Invoice Queries > Raise Invoice Query** (top right hand corner of the page)
+
+If you have any questions relating to billing or have an account query please call our Accounts team on **0845 458 3535** or email accounts@ukfast.co.uk.
+
```

Comment:
If they have queries on billing they should use the Query form above ideally, for a direct trackable line into accounts. 
Again prefer to keep emails off the internet :)